app/retail/emails.py

```diff
@@ -685,25 +685,22 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from external_bounties.models import ExternalBounty
-    subject = "In Pursuit Of Open Source Financial Freemdom"
-    new_kudos_pks = [528, 588, 589]
+    subject = "Introducing Gitcoin Ambassadors | MetaTX Grows"
+    new_kudos_pks = [486, 485, 484]
```

Comment:
@vivek @mark 

http://bits.owocki.com/d464d2891925/Screen%20Shot%202018-12-05%20at%202.28.37%20PM.png

the ‘new kudos that launched this week’ are not in fact new at all. they’re the same ones we emailed out two weeks ago